### ISSUE TITLE
pyftpdlib.test.test_functional.TestCallbacks.test_on_incomplete_file_received test fails

### ISSUE THREAD

**Issue by DerDakon** [giampaolo/pyftpdlib#471]
I have observed this with 1.5.3 and 1.5.4 on sparc, there is also a report for amd64, see https://bugs.gentoo.org/636410

**Comment by giampaolo** [comment]
Yes this is a known issue. It's the test which is instable: the underlying functionality is fine.

**Comment by sbraz** [comment]
@giampaolo in this case could you make failure non-fatal? If I understand correctly, pytest seems to have `xfail(strict=False)` for such tests: https://docs.pytest.org/en/latest/skipping.html

**Comment by giampaolo** [comment]
Mmm am afraid not. This is not the only unstable test. They have to be rewritten/rethinked. The problem is that threads make these failures unpredictable. I think a way to solve that would be to use subprocesses.
